### PR TITLE
chore: make sure spectral is installed

### PR DESCRIPTION
**Pull request by thinkingserious**
### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Review by JenniferMah — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 install:
 	npm install
 
-spectral:
+spectral: install
```
Comment (childish-sambino) [inline Makefile:6]: Why is this needed? I thought librarian does a `make install` as part of the `init` step.
